packages/create-app/src/commands/init.ts

```diff
@@ -0,0 +1,28 @@
+import {execSync} from 'child_process';
+
+import {Command, Flags} from '@oclif/core';
+import path from 'pathe';
+
+export default class InitCommand extends Command {
+  static description = 'Create new GestaltJS app';
+  static flags = {
+    name: Flags.string({
+      char: 'n',
+      description: 'project name',
+    }),
+    path: Flags.string({
+      char: 'p',
+      description: 'project path',
+    }),
+  };
+
+  async run(): Promise<void> {
+    const {args, flags} = await this.parse(InitCommand);
+    const name = flags.name ? flags.name : 'my-gestalt-project';
```

Comment:
What do you think about making `name` a required [argument](https://oclif.io/docs/args) instead of a flag?
```
create-app app-name
```